src/core/tests/widgets/test_icon.py

```diff
@@ -11,10 +12,28 @@ def setUp(self):
         self.factory.Icon = MagicMock(return_value=MagicMock(spec=toga_dummy.factory.Icon))
 
         self.test_path = "Example.bmp"
+        self.system = False
```

Comment:
What is the purpose of this line?

Reply:
I was setting self.system to True and False to test for the instances where an icon is in the resources directory or not.

Reply:
Yes, but the value isn't actually *used* anywhere.

Reply:
Yes, that's correct. The value of self.system is set to False by default. Hence, there was no need to explicitly add that line. It can be removed.